content/en/docs/concepts/services-networking/service.md

```diff
@@ -1072,6 +1072,9 @@ in those modified security groups.
 
 {{< /note >}}
 
+Further documentation on assigining Elastic IPs and other common use-cases may be found
+on the [AWS EKS NLB documentation pages](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html).
```

Comment:
This seems OK. Really, we should aim to move details of specific cloud provider implementations into the Reference section of this website (see https://github.com/kubernetes/website/issues/14770).